INZFS.MVC/Views/FundApplication/_StaticPage.cshtml

```diff
@@ -0,0 +1,47 @@
+@model StaticPageModel
+@using INZFS.MVC.Models.DynamicForm
+
+@if (Model.PageName == "subsidy-requirements")
```

Comment:
convert this page to lowercase please .

@if (Model.PageName.ToLower() == "subsidy-requirements")

or

var pageName = Model.PageName.ToLower() ;
@if (pageName  == "subsidy-requirements")

Reply:
Done